module/signature/invalid_threshold.go

```diff
@@ -0,0 +1,32 @@
+// +build relic
```

Comment:
I was wondering if using the usual `ThresholdProvider` would be enough. We could just update the `Sign` function to test if the private key is `nil` or not. 
The `reconstruct` function doesn't have to be updated.